SECURITY.md

```diff
@@ -0,0 +1,116 @@
+This is the Security Policy for the Perl Imager distribution.
+
+Security vulnerabilities can be reported via the project GitHub
+repository Security Advisories. (If you do not have access to GitHub,
+then you can report issues via email to tony@develop-help.com.)
```

Comment:
I would link to https://github.com/tonycoz/imager/security/advisories/new instead